MAUI/TabView/Header-Display-Mode.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Header Display Mode in Maui Tabbed View control | Syncfusion
```

Comment:
Header display mode in .NET MAUI Tab View control

Reply:
Updated the title.